homeassistant/components/integration/sensor.py

```diff
@@ -112,9 +120,12 @@ def __init__(
             self._unit_of_measurement = None
         else:
             self._unit_of_measurement = unit_of_measurement
+            self._set_attributes()
```

Comment:
? Where is this one?

Reply:
Ohh, was part of an earlier version. Will remove